packages/api/src/external/carequality/document/process-ihe-to-gw-document-retrieval.ts

```diff
@@ -2,50 +2,73 @@ import { out } from "@metriport/core/util/log";
 import { Bundle, BundleEntry } from "@medplum/fhirtypes";
 import { errorToString } from "@metriport/core/util/error/shared";
 import { capture } from "@metriport/core/util/notifications";
-import { DocumentReference } from "@metriport/ihe-gateway-sdk";
+import { DocumentReference, DocumentRetrievalRespFromExternalGW } from "@metriport/ihe-gateway-sdk";
 import { MedicalDataSource } from "@metriport/core/external/index";
-import { DocumentRetrievalResult } from "../document-retrieval-result";
 import { isConvertible } from "../../fhir-converter/converter";
-import { setDocQueryProgressWithSource } from "../../hie/set-doc-query-progress-with-source";
-import { tallyDocQueryProgressWithSource } from "../../hie/tally-doc-query-progress-with-source";
+import { setDocQueryProgress } from "../../hie/set-doc-query-progress";
+import { tallyDocQueryProgress } from "../../hie/tally-doc-query-progress";
 import { convertCDAToFHIR } from "../../fhir-converter/converter";
 import { upsertDocumentsToFHIRServer } from "../../fhir/document/save-document-reference";
-import { cqToFHIR } from "../../fhir/document";
+import { cqToFHIR } from "./shared";
 import { getDocuments } from "../../fhir/document/get-documents";
 import { ingestIntoSearchEngine } from "../../aws/opensearch";
 
-export async function processDocumentRetrievalResults({
+export async function processIHEToExternalGwDocumentRetrievals({
```

Comment:
Likewise, can we call this `processOutboundDocumentRetrievalResponses`? I think adding "results" helps indicate this is after the response from the external GWs.